src/components/About/About.jsx

```diff
@@ -0,0 +1,13 @@
+import "./About.css";
+
+const About = () => {
+  return (
+    <div className="about">
+      You&#39;re on the Author page.
+      <br />
```

Comment:
` <br />` shouldn't be used in the text because it can't be styled, and it's diffictult to update the text correctly. You should use 2 `p` tags if you want 2 texts one under another